### PR TITLE
:heavy_plus_sign: Add django-log-outgoing-requests

### LINKED ISSUE #344: Add log-outgoing-requests to Objects API
During debugging an Objects API - Open Notificaties issue Laurens and I noticed we don't have https://github.com/maykinmedia/django-log-outgoing-requests installed in the project. Please do so, it should give proper insight into what the Objects API is doing (both to OT API and ON)

### PR DESCRIPTION
**Pull request by stevenbal**
Fixes #344
replacement for https://github.com/maykinmedia/objects-api/pull/345

### CODE REVIEW

**Review by annashamray — CHANGES_REQUESTED** [review]
1. Let's have a global var LOG_REQUESTS which switches off and on the logging. You can take a look at the config of open zaak or open forms 
2. Let's turn off saving to the DB by default: 
```
LOG_OUTGOING_REQUESTS_DB_SAVE = config("LOG_OUTGOING_REQUESTS_DB_SAVE", default=False)
```

File: src/objects/conf/base.py [hunk 1]
```diff
@@ -231,6 +233,8 @@
 FIXTURE_DIRS = (os.path.join(DJANGO_PROJECT_DIR, "fixtures"),)
 
 LOGGING_DIR = os.path.join(BASE_DIR, "log")
+LOG_REQUESTS = config("LOG_REQUESTS", default=False)
+LOG_OUTGOING_REQUESTS_DB_SAVE = config("LOG_OUTGOING_REQUESTS_DB_SAVE", default=True)
```
Comment (annashamray) [inline src/objects/conf/base.py:237]: Please add new env vars to the docs

**Review by annashamray — COMMENTED** [review]
Looks good. 
Could you just rebase and document new env vars? 

Btw is it planned to add django-log-outgoing-requests to open-api-framework?

**Comment by stevenbal** [comment]
@annashamray there is an issue for it https://github.com/maykinmedia/open-api-framework/issues/14 (and also for setup_configuration https://github.com/maykinmedia/open-api-framework/issues/15)

File: src/objects/conf/base.py (hunk 1 above)
Comment (Coperh) [reply src/objects/conf/base.py]: Added to `docs/installation/config.rst`

**Review by annashamray — APPROVED** [review]

**Pull request merged**